### PR TITLE
#3: Estimation training

### LINKED ISSUE #3: Design estimation training
## Situation

We need a training for estimation methods in agile projects.

## Acceptance Criteria

1. Covers multiple estimation methods
2. Explains how to reduce uncertainty
3. Contains interactive exercises

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #3.

This PR contains a presentation document (Libre Office Impress) about agile estimation methods including training exercises.

Update:

I realized that I forgot to demonstrate relative estimation on the example of complexity points.

### CODE REVIEW

**Comment by tkilias** [comment]
Slide 14: Median is not 10.5, Median needs to be a number of the set.

**Comment by exaSR** [comment]
> Slide 14: Median is not 10.5, Median needs to be a number of the set.

Not true, if the set has an even number of members, the median is defined as the average of the two "middle" elements :)

File: estimating/estimating.odp
Comment (ckunki) [inline estimating/estimating.odp:1]: One important take-away for me is that using person-days as unit also constantly raises the recurring question how to convert effort (how much capacity or billed hours does it cost?) into ETA (= which point in time will the task be finished?) and vice versa.

All this combined with the limited accuracy in comparisson to the required effort for creating a decent estimation in my eyes makes strongly limits the benefit of estimations.

In result I would strongly vote to 1. reduce the effort for estimations as much as possible (e.g. by doing a naive lightning guess) and 2. to rather update estimations frequently whenever additional information is available.

**Comment by tkilias** [comment]
> > Slide 14: Median is not 10.5, Median needs to be a number of the set.
> 
> Not true: If the set has an even number of members, the median is defined as the average of the two "middle" elements :)

Ah, you are right, I remember this now, didn't check the number of the elements

File: estimating/estimating.odp
Comment (ckunki) [reply estimating/estimating.odp:1]: I first misinterpreted slide "Group estimations" in the direction that it wants to recommend the reader to arrange multiple estimations into groups. 

Only later on I realized that the slide rather wants to introduce methods for creating estimations by a group of people. Maybe we could improve the title or add an icon or a sub title?

Comment (ckunki) [reply estimating/estimating.odp:1]: In general aside from a training an activity like estimating in my eyes could best be supported in a learning-by-doing manner with daily coaching,

Comment (redcatbear) [reply estimating/estimating.odp:1]: I agree that practice makes perfect. And that the more practice you have, the quicker the whole estimation will go.

Comment (redcatbear) [reply estimating/estimating.odp:1]: Changed "Group estimations" to "Estimating together".

[the remark this answers is not in the document]
Comment (tkilias) [reply estimating/estimating.odp:1]: slide 28, spikes animation seems broken

**Review by tkilias — APPROVED** [review]

**Pull request merged**